### ISSUE TITLE
gitlab directories don't work

### ISSUE THREAD

**Issue by RazerM** [dependabot/dependabot-core#1127]
I'm trying to use dependabot-script. I printed `fetcher.repo_contents` to see why directories weren't being search for requirements files, and I think this is the reason:

```text
[#<OpenStruct name="requirements", path="requirements", type="tree", size=0>, #<OpenStruct name="setup.py", path="setup.py", type="file", size=0>]
```

The file_fetcher for Python looks for `type == "dir"`, but the `requirements` directory has `type = "tree"`

I'm not a ruby developer but I guess this is meant to translate the type to what dependabot expects:

https://github.com/dependabot/dependabot-core/blob/a69c17a7419cc3b8558eb5f2e2a24105e9479913/common/lib/dependabot/file_fetchers/base.rb#L207

**Comment by greysteil** [comment]
Ah, interesting. Using with GitLab?

**Comment by RazerM** [comment]
yes

**Comment by greysteil** [comment]
Should be fixed in `0.106.17`, which I'm pushing now - will update the script repo.

Thanks for the bug report!

**Comment by RazerM** [comment]
It works, thanks!
